index.html

```diff
@@ -0,0 +1,268 @@
+      <li> <button class="btn" type="button">Ruby</button></li>
+    </ul>
+  </div>
+  <div class="see-project-1">
+    <a class="project" href="#"> see project</a>
+  </div>
+</div>
+
+<div class="profesional-art-1">
+  <div class="H3">
+    <h2> Profesional Art Printing Data</h2>
+  </div>
+  <div class="p2">
+    <p>A daily selection of privately personalized reads;
+      no accounts or sign-ups required.
+      has been the industry's standard</p>
+  </div>
+  <div class="button-0">
+    <ul class="buttons-0">
+      <li> <button class="btn" type="button">HTML</button></li>
+      <li> <button class="btn" type="button">Bootstrap</button></li>
+      <li> <button class="btn" type="button">Ruby</button></li>
+    </ul>
+  </div>
+  <div class="see-project-1">
+    <a class="project" href="#"> see project</a>
+  </div>
+</div>
+
+<div class="profesional-art-1">
+  <div class="H3">
+    <h2> Profesional Art Printing Data</h2>
+  </div>
+  <div class="p2">
+    <p>A daily selection of privately personalized reads;
+      no accounts or sign-ups required.
+      has been the industry's standard</p>
+  </div>
+  <div class="button-0">
+    <ul class="buttons-0">
+      <li> <button class="btn" type="button">HTML</button></li>
+      <li> <button class="btn" type="button">Bootstrap</button></li>
+      <li> <button class="btn" type="button">Ruby</button></li>
+    </ul>
+  </div>
+  <div class="see-project-1">
+    <a class="project" href="#"> see project</a>
+  </div>
+</div>
+
+<div class="profesional-art-1">
+  <div class="H3">
+    <h2> Profesional Art Printing Data</h2>
+  </div>
+  <div class="p2">
+    <p>A daily selection of privately personalized reads;
+      no accounts or sign-ups required.
+      has been the industry's standard</p>
+  </div>
+  <div class="button-0">
+    <ul class="buttons-0">
+      <li> <button class="btn" type="button">HTML</button></li>
+      <li> <button class="btn" type="button">Bootstrap</button></li>
+      <li> <button class="btn" type="button">Ruby</button></li>
+    </ul>
+  </div>
+  <div class="see-project-1">
+    <a class="project" href="#"> see project</a>
+  </div>
+</div>
+
+<div class="profesional-art-1">
+  <div class="H3">
+    <h2> Profesional Art Printing Data</h2>
+  </div>
+  <div class="p2">
+    <p>A daily selection of privately personalized reads;
+      no accounts or sign-ups required.
+      has been the industry's standard</p>
+  </div>
+  <div class="button-0">
+    <ul  class="buttons-0">
+      <li> <button class="btn" type="button">HTML</button></li>
+      <li> <button class="btn" type="button">Bootstrap</button></li>
+      <li> <button class="btn" type="button">Ruby</button></li>
+    </ul>
+  </div>
+  <div class="see-project-1">
+    <a class="project" href="#"> see project</a>
+  </div>
+</div>
+        
+     </div>
+   </section>
+
+   <section class="section-3">
+     <div class="about-me">
+     <div class="me">
+       <h2> About me </h2>
+     </div>
```

Comment:
- [ ] Kindly change the text width when it's on the mobile version to be smaller and not overlap the paragraph.

![image](https://user-images.githubusercontent.com/35535130/166128064-9fa8f713-5a22-4a97-8491-f1bf52e968c3.png)